hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerMetrics.java

```diff
@@ -44,6 +44,10 @@ public final class ContainerBalancerMetrics {
       "in the latest iteration.")
   private MutableCounterLong numContainerMovesInLatestIteration;
 
+  @Metric(about = "Number of timeout container moves performed by " +
+      "Container Balancer in the latest iteration.")
+  private MutableCounterLong numTimeoutContainerMovesInLatestIteration;
```

Comment:
Can we rename this to `numContainerMovesTimeoutInLatestIteration` and `numContainerMovesCompletedInLatestIteration`?